contracts/non-fungible-tokens/andromeda-cw721-offers/src/contract.rs

```diff
@@ -114,7 +114,7 @@ fn execute_place_offer(
     let coin: &Coin = &info.funds[0];
     require(
         // TODO: Add support for other denoms later.
-        coin.denom == "uusd",
+        coin.denom == "uusd" || coin.denom == "ujunox",
```

Comment:
Should be ujuno I think, ujunox is the testnet version.

Reply:
On second thought, we should probably just provide the denom in the instantiate.

Reply:
Added `valid_denoms` to instantiate. Also added a test for it.

Reply:
I just realized that I dont think we can accept multiple denoms. This is because only a single offer is allowed and it checks that the amount is greater than the previous offer. With multiple denoms this isn't possible without use of price oracles.

Reply:
That's a valid point. Think the only solution currently is to cut it down to a single denom. I'll do that today.